release/archives/linux/data-prepper-jdk-x64.sh

```diff
@@ -5,6 +5,25 @@
 # SPDX-License-Identifier: Apache-2.0
 #
 
+if [[ $# == 0 ]]; then
+    echo "Reading pipelines and data-prepper configuration files from Data Prepper home directory."
+elif [[ $# == 2 ]]; then
+    echo
+    echo "Data Prepper now supports reading pipeline and data-prepper configuration files"
+    echo "from Data Prepper home directory automatically."
+    echo "You can continue to specify paths to configuration files as command line arguments,"
+    echo "but that support will be dropped in a future release."
+    echo
+else
```

Comment:
I think we accept single argument as well.

Reply:
Do we in 1.x? I think the user must supply both pipeline and configuration files.

Reply:
> Do we in 1.x? I think the user must supply both pipeline and configuration files.

No we don't. But we just need to clean up `DataPrepperArgs` class.